app/controllers/releases_controller.rb

```diff
@@ -0,0 +1,76 @@
+class ReleasesController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound, with: :catch_not_found
+  before_action :set_release, only: [:edit, :update, :show, :destroy]
+
+  def index
+    @releases = Release.all.order(created_at: :desc)
```

Comment:
They should be ordered by `released_on`. We may not make them in order so `released_on` is more reliable for order

Reply:
Makes sense. Thanks.

Reply:
Good catch.